tests/unit/test_pokemon_selector.py

```
@@ -0,0 +1,108 @@
+"""
+Unit tests for the PokemonSelector class.
+"""
+import pytest
+from unittest.mock import MagicMock
```

Ruff (0.8.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)